src/components/course/course-header/data/hooks/useCourseRunCardData.js

```diff
@@ -28,6 +33,19 @@ const useCourseRunCardData = ({
   } = courseRun;
   const isCourseRunCurrent = availability === COURSE_AVAILABILITY_MAP.CURRENT;
   const isUserEnrolled = !!userEnrollment;
+  const { enterpriseConfig } = useContext(AppContext);
+  const {
+    state: {
+      activeCourseRun: {
+        seats,
+      },
+    },
+  } = useContext(CourseContext);
+  const seat = useMemo(
+    () => findHighestLevelSeatSku(seats),
+    [seats],
+  );
+  const enrollmentUrl = getExecEdEnrollUrl(enterpriseConfig.slug, courseRun.courseUuid, seat?.sku);
```

Comment:
nit: consider using an object param as it makes it a bit more obvious what's being passed in by the caller, kind of the kwargs in Python.

Reply:
Good call, I had one argument mix-up with it myself.